src/theory/bv/rewrites

```diff
@@ -6,328 +6,882 @@
 ; extracts the range [i..j]. Vide https://smtlib.cs.uiowa.edu/theories-FixedSizeBitVectors.shtml
 ; Also concat is big-endian.
 
-(define-rule* ConcatFlatten
-  ((xs ?BitVec :list)
-   (s ?BitVec)
-   (ys ?BitVec :list)
-   (zs ?BitVec :list))
-  (concat xs (concat s ys) zs)
-  (concat xs s ys zs))
-(define-rule ConcatExtractMerge
-  ((xs ?BitVec :list)
-   (s ?BitVec)
-   (ys ?BitVec :list)
-   (i Int) (j Int) (k Int)
-  )
-  (concat xs (extract k (+ j 1) s) (extract j i s) ys)
-  (concat xs (extract k i s) ys))
+(define-rule* bv-concat-flatten
+	((xs ?BitVec :list)
```

Comment:
There are too many spaces before the expressions. Please reformat so that it is analogous to e.g., https://github.com/cvc5/cvc5/blob/proof-new/src/theory/strings/rewrites

Reply:
Yes, sounds good